src/components/HistoryTable/row/DateSeparator.tsx

```diff
@@ -0,0 +1,50 @@
+import { CSSProperties } from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import { format, utcToZonedTime } from "date-fns-tz";
+import { useUserTimeZone } from "hooks/useUserTimeZone";
+
+const { gray } = uiColors;
+interface DateSeparatorProps {
+  style: CSSProperties;
+  date: Date;
+}
+
+export const DateSeparator: React.FC<DateSeparatorProps> = ({
+  style,
+  date,
+}) => {
+  const tz = useUserTimeZone();
```

Comment:
Can you look at the network activity as we scroll. Im worried this  might get called allot as the user scrolls. Since react-window will continuously rerender this component.

Reply:
If it does we should either pass in a flag to use a cached value. Or pass this value from a higher level component that doesn't rerender often.